doc/docs.md

```diff
@@ -1905,6 +1905,61 @@ fn announce(s Speaker) {
 ```
 For more information, see [Dynamic casts](#dynamic-casts).
 
+```v
+struct Cat {}
+
+interface Animal {}
+
+fn (a Animal) is_animal() {
+	println('is an Animal.')
+}
+
+fn new_cat() Animal {
+	return &Cat{}
```

Comment:
would not just `return Cat{}` work?

Reply:
iirc, that would return an interface instance with a pointer to stack memory, which will be invalid when the function returns 😐

Reply:
there are still quite a few codegen bugs when casting things to interfaces unfortunately.

Reply:
This example still compiles and runs correctly in the way spytheman describes. I've just changed it that way for simplicity.